internal/pkg/bootstrap/handlers/database.go

```diff
@@ -13,16 +13,16 @@ import (
 
 	"github.com/edgexfoundry/edgex-go/internal/pkg/bootstrap/interfaces"
 	"github.com/edgexfoundry/edgex-go/internal/pkg/db"
-	"github.com/edgexfoundry/edgex-go/internal/pkg/v2/infrastructure/redis"
-	v2Interface "github.com/edgexfoundry/edgex-go/internal/pkg/v2/interfaces"
+	"github.com/edgexfoundry/edgex-go/internal/pkg/infrastructure/redis"
+	v2Interface "github.com/edgexfoundry/edgex-go/internal/pkg/interfaces"
```

Comment:
remove alias and change add alias for `pkg/bootstrap/interfaces` as `bootstrapInterfaces`

Reply:
Modified.